File: 1. Two Sum/two_sum.go
func twoSum(nums []int, target int) []int {
    var x int
    for i, num1 := range nums {
        diff := target - num1
        found := false
        
        for j, num2 := range nums[i+1:] {
            if num2 == diff {
                found = true
                x = j + i + 1
            } 
        }

        if found == true {
            return []int{i,x}
        }
    }
    return []int{}
}

File: 1. Two Sum/two_sum.py
class Solution(object):
    def twoSum(self, nums, target):
        """
        :type nums: List[int]
        :type target: int
        :rtype: List[int]
        """
        complement = {}
        for pos, num in enumerate(nums):
            if target - num in complement:
                return [complement[target - num], pos]
            else:
                complement[num] = pos
